main.tf

```diff
@@ -88,6 +88,8 @@ module "nat_instance" {
   nat_instance_type     = var.nat_instance_type
   public_subnets_ids    = module.vpc.public_subnets
   private_subnets_ids   = module.vpc.private_subnets
+  create_test_instance  = var.create_test_instance
```

Comment:
what the point ? just spinning an ec2 instance in the private subnet ? how is it relevant to NAT behavior ?

Reply:
What do you mean by "not the place for it"?
"create_test_instance" is an optional attribute that you can pass to vpc module.
For cases that you want to make sure that your traffic from private network is properly routed through  nat instances's ENI. (for example when we had an issue with BG deployment of pulling images from ecr because it's a lot more difficult to check it from ECS on fargate)

Reply:
it means that I understand the purpose of dummy ec2 instance for debugging. I just don't see why it need to be part of a vpc module. I dont want to make a code change and run terraform apply and change the state for it. I can simply launch a test instance manually (or some dedicated automation if you'd like).

Reply:
Kind of agreed :)